libs/sdk-ui-web-components/src/visualizations/Insight.tsx

```diff
@@ -0,0 +1,59 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+import invariant from "ts-invariant";
+
+import { IAnalyticalBackend } from "@gooddata/sdk-backend-spi";
+import { GoodDataSdkError, ILoadingState, IDrillEvent, IExportFunction, ILocale } from "@gooddata/sdk-ui";
+import { IInsight } from "@gooddata/sdk-model";
+import {
+    CustomElementAdapter,
+    LOAD_COMPONENT,
+    EVENT_HANDLER,
+    GET_VISUALIZATION,
+} from "./CustomElementAdapter";
+
+type IInsightView = typeof import("@gooddata/sdk-ui-ext/esm/insightView/InsightView").InsightView;
```

Comment:
Can we do
```ts
import type { InsightView } from "@gooddata/sdk-ui-ext";
```
instead? That should also make the dependency cruiser checks that are currently failing pass.